Backend/app/rider-platform/rider-app/Main/src-read-only/API/Action/Dashboard/RideBooking/Select.hs

```diff
@@ -0,0 +1,38 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module API.Action.Dashboard.RideBooking.Select
+  ( API.Types.Dashboard.RideBooking.Select.API,
+    handler,
+  )
+where
+
+import qualified API.Types.Dashboard.RideBooking.Select
+import qualified Domain.Action.Dashboard.RideBooking.Select as Domain.Action.Dashboard.RideBooking.Select
+import qualified "this" Domain.Action.UI.Select
+import qualified "this" Domain.Types.Estimate
+import qualified Domain.Types.Merchant
+import qualified "this" Domain.Types.Person
+import qualified Environment
+import EulerHS.Prelude
+import qualified Kernel.Types.APISuccess
+import qualified Kernel.Types.Beckn.Context
+import qualified Kernel.Types.Id
+import Kernel.Utils.Common
+import Servant
+import Tools.Auth
+
+handler :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Environment.FlowServer API.Types.Dashboard.RideBooking.Select.API)
+handler merchantId city = postSelectEstimateSelect merchantId city :<|> getSelectQuotes merchantId city :<|> getSelectResult merchantId city :<|> postSelectCancel merchantId city
+
+postSelectEstimateSelect :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Kernel.Types.Id.Id Domain.Types.Person.Person -> Kernel.Types.Id.Id Domain.Types.Estimate.Estimate -> Domain.Action.UI.Select.DSelectReq -> Environment.FlowHandler Kernel.Types.APISuccess.APISuccess)
+postSelectEstimateSelect a5 a4 a3 a2 a1 = withFlowHandlerAPI $ Domain.Action.Dashboard.RideBooking.Select.postSelectEstimateSelect a5 a4 a3 a2 a1
+
+getSelectQuotes :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Kernel.Types.Id.Id Domain.Types.Person.Person -> Kernel.Types.Id.Id Domain.Types.Estimate.Estimate -> Environment.FlowHandler Domain.Action.UI.Select.SelectListRes)
+getSelectQuotes a4 a3 a2 a1 = withFlowHandlerAPI $ Domain.Action.Dashboard.RideBooking.Select.getSelectQuotes a4 a3 a2 a1
```

Comment:
manual code:

```
callSelectList :: ShortId DM.Merchant -> Id DP.Person -> Id DEstimate.Estimate -> FlowHandler DSelect.SelectListRes
callSelectList merchantId personId estimate = do
  m <- withDashboardFlowHandlerAPI $ findMerchantByShortId merchantId
  US.selectList (personId, m.id) estimate
```

`US.selectList` used `withFlowHandlerAPI` wrapper, in generated code we have `withDashboardFlowHandlerAPI`